R/database-snomed.R

```diff
@@ -0,0 +1,536 @@
+#' Add SNOMED tables
+#'
+#' Creates or updates a SNOMED CT database
+#'
+#' @param path_destination Character string. Path where the database files
+#'   will be written.
+#' @param release_index Integer. Which TRUD release index to use (1 = latest).
+#'
+#' @return Invisible `TRUE` on success
+#'
+add_snomed_tables <- function(
+  path_destination = ".",
+  release_index = 1
```

Comment:
```suggestion
  item_id = 1799,
  release_index = 1
```

See previous comment.